src/viewProviders/flinkStatements.ts

```diff
@@ -35,30 +35,28 @@ export class FlinkStatementsViewProvider
   /**
    * (Re)paint the view. If forceDeepRefresh=true, then will force a deep fetch of the statements.
    */
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  async refresh(forceDeepRefresh: boolean = false): Promise<void> {
+  async refresh(): Promise<void> {
     // Out with any existing subjects.
     this.resourcesInTreeView.clear();
 
     if (this.resource !== null) {
       const loader = ResourceLoader.getInstance(this.resource.connectionId) as CCloudResourceLoader;
 
-      // Fetch statements using the loader, pushing down need to do deep refresh.
-      const statements: FlinkStatement[] = await window.withProgress(
-        {
-          location: { viewId: this.viewId },
-          title: "Loading Flink statements...",
-        },
+      void this.withProgress(
```

Comment:
No need to await this if the inner awaitable function does all of the work with the fetched statements. The prior spelling returning the statements out from `window.withProgress()` was unnecessarily complicated and also didn't properly revert the view to empty state while the fetching was happening.